src/components/Filter/FilterComponent.jsx

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import { useFilter } from "../../contexts/filter-context";
+import "../Filter/filtercomponent.css";
+const FilterComponent = ({ items }) => {
+  const { state, productDispatch } = useFilter();
+  const { sortBy, price, rating, categories } = state;
+  const { console, controller, games, accessories } = categories;
+  return (
+    <div>
```

Comment:
since you are using `aside` extra div outside is not required. alternatively, you can use Fragments if required.